kafka-connect-mqtt/src/main/scala/com/datamountaineer/streamreactor/connect/mqtt/config/MqttSourceConfig.scala

```diff
@@ -0,0 +1,128 @@
+/**
+  * Copyright 2016 Datamountaineer.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  **/
+
+package com.datamountaineer.streamreactor.connect.mqtt.config
+
+import java.util
+
+import org.apache.kafka.common.config.ConfigDef.{Importance, Type}
+import org.apache.kafka.common.config.{AbstractConfig, ConfigDef}
+
+/**
+  * Created by andrew@datamountaineer.com on 22/09/16. 
+  * stream-reactor
+  */
+object MqttSourceConfig {
```

Comment:
I think we should have the `connect.` prefix on the options. For example with the confluent dockers they have functions to handle the mapping of enviroment variable to anything being with `connect`